Dockerfile

```
@@ -1,63 +1,112 @@
 FROM node:14-buster-slim as dep-builder
```

```suggestion
FROM node:16-buster-slim as dep-builder
```
Please also bump it to node 16. 

rsshub.app runs on node 16. `.nvmrc` also points to node 16.
This should help to keep routes behaviour consistent across different environments.

---

Sure. What about also bumping Debian to Bullseye? Buster will become LTS in July and only receive security updates from Debian LTS Team instead of Debian Security Team.